melodic/distribution.yaml

```diff
@@ -15510,6 +15510,24 @@ repositories:
       url: https://github.com/ElettraSciComp/witmotion_IMU_ros.git
       version: main
     status: maintained
+  witmotion_ros-release:
+    doc:
+      type: git
+      url: https://github.com/ElettraSciComp/witmotion_IMU_ros.git
+      version: main
+    release:
+      packages:
+      - witmotion_ros
```

Comment:
This doesn't seem right for a repository with only one package at the top-level.  These types of repositories typically have rosdistro entries that look like https://github.com/ros/rosdistro/blob/6ba20049a926757b8450cf35b4602cb2ad364e63/melodic/distribution.yaml#L122-L136

Reply:
Fixed with [59a67b7](https://github.com/ros/rosdistro/pull/36427/commits/59a67b778cc2bd5c4fabd87dfb15afe9d7273eb3)